dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.connectors-2.1.internal.feature

```diff
@@ -20,6 +20,6 @@ IBM-API-Package: \
   com.ibm.ws.jca.jakarta, \
   com.ibm.ws.jca.utils.jakarta, \
   com.ibm.ws.jca.feature.jakarta
-kind=noship
-edition=full
+kind=beta
+edition=base
```

Comment:
I believe when we beta these features they should go into core ediition.  Then when we GA we will differentiate between core/base.
This should match the edition where the API is currently: https://github.com/OpenLiberty/open-liberty/blob/79db86cbd955818cb3b0353787b8f53bc4d4155e/dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.connectors-2.1.feature#L6-L7

Reply:
In this case, the private API feature is not a precedent for what to do because it was likely put into core for the sake of other core features that rely on it (connection management, jdbc) rather than because the corresponding public feature for connectors ought to be core for beta.  According to the documentation https://github.ibm.com/websphere/WS-CD-Open/wiki/Writing-features the edition for beta should match the edition of the previous GA features, which for connectors-2.0 is base.  So I think Tim's change is correct as written.

Reply:
2.0.internal feature is in base, so the 2.1.internal should also be in base.  There is not feature in core that depends on it.  In general core = web profile and base = full platform with some exceptions.

Reply:
@tevans78 just asked me to double check and it appears I was looking at the wrong feature yesterday when I though that `/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.connectors-2.0.internal.feature` was listed as core.  Now when I check, it very clearly shows up as base, which means no change was needed here after all,
```
kind=ga
edition=base
```

Reply:
In general when we beta features, we put them in the edition they will be when they ga.  For Open Liberty, core and base do not have a meaning because there isn't an Open Liberty core or base install option.  That offering is a WebSphere Liberty concept and that is why we need to do the right things so that the right things are included for WebSphere Liberty when we do move features to ga.